.travis.yml

```diff
@@ -54,7 +54,7 @@ before_install:
   # Install Java goodies
   - mkdir java-files
   # Install PhyloNet
-  - wget http://bioinfo.cs.rice.edu/sites/g/files/bxs266/f/kcfinder/files/PhyloNet_3.5.7.jar -O java-files/PhyloNet_3.5.7.jar
+  # wget http://bioinfo.cs.rice.edu/sites/g/files/bxs266/f/kcfinder/files/PhyloNet_3.5.7.jar -O java-files/PhyloNet_3.5.7.jar
```

Comment:
We removed it because of new issues regarding certificates and will install with conda later ([new issue](https://github.com/biocore/WGS-HGT/issues/33)). However, adding the flag ``` --no-check-certificate``` temporarily resolves the certificate issue, @qiyunzhu could you uncomment the install command and add that flag?